scripts/aws/cfn/microservice.yml

```diff
@@ -472,7 +472,7 @@ Resources:
         ContainerEnvEntry42: !Sub 'PnSignProviderSwitch=${PnSignProviderSwitch}'
         ContainerEnvEntry43: !Sub 'PnSsSignRetryStrategyMaxAttempts=${PnSsSignRetryStrategyMaxAttempts}'
         ContainerEnvEntry44: !Sub 'PnSsSignRetryStrategyMinBackoff=${PnSsSignRetryStrategyMinBackoff}'
-        ContainerEnvEntry45: !Sub 'PnSsNamirialServerAddress=${PnSsNamirialServerAddress}'
+        ContainerEnvEntry45: !Sub 'PnEcNamirialServerAddress=${PnEcNamirialServerAddress}'
```

Comment:
PnEcNamirialServerAddress?

Reply:
Si, attualmente la libreria pn-ss-namirial-sign prevede l'utilizzo della property "PnEcNamirialServerAddress".

Dal README di pn-ss-namirial-sign:

The library uses the following environment variables:
- `PnEcNamirialServerAddress`: the URL of the Namirial Sign service

Reply:
Non è più così